cmd/collector/main.go

```diff
@@ -6,71 +6,54 @@ import (
 	"os"
 	"time"
 
+	"github.com/pokt-foundation/utils-go/environment"
 	logger "github.com/sirupsen/logrus"
 
-	"github.com/adshmh/meter/cmd"
 	"github.com/adshmh/meter/collector"
 	"github.com/adshmh/meter/db"
 )
 
-const (
-	COLLECT_INTERVAL_DEFAULT_SECONDS = 300
-	REPORT_INTERVAL_DEFAULT_SECONDS  = 30
-	MAX_ARCHIVE_AGE_DEFAULT_DAYS     = 30
-
-	ENV_COLLECT_INTERVAL_SECONDS = "COLLECTION_INTERVAL_SECONDS"
-	ENV_REPORT_INTERVAL_SECONDS  = "REPORT_INTERVAL_SECONDS"
-	ENV_MAX_ARCHIVE_AGE_DAYS     = "MAX_ARCHIVE_AGE"
+var (
+	influxDBURL           = environment.MustGetString("INFLUXDB_URL")
+	influxDBToken         = environment.MustGetString("INFLUXDB_TOKEN")
+	influxDBOrg           = environment.MustGetString("INFLUXDB_ORG")
+	influxDBBucketDaily   = environment.MustGetString("INFLUXDB_BUCKET_DAILY")
+	influxDBBucketCurrent = environment.MustGetString("INFLUXDB_BUCKET_CURRENT")
+
+	postgresUser     = environment.MustGetString("POSTGRES_USER")
+	postgresPassword = environment.MustGetString("POSTGRES_PASSWORD")
+	postgresDB       = environment.MustGetString("POSTGRES_DB")
+	postgresHost     = environment.MustGetString("POSTGRES_HOST")
+
+	collectionInterval = int(environment.GetInt64("COLLECTION_INTERVAL_SECONDS", 300))
+	reportingInterval  = int(environment.GetInt64("REPORT_INTERVAL_SECONDS", 30))
+	maxArchiveAgeDays  = int(environment.GetInt64("MAX_ARCHIVE_AGE", 30))
```

Comment:
Same as previous comments: OK to use a separate package/lib to handle the env. loading.
However, this collection should be:
- using constant declared at the top for readability
- using options struct to group them, for readability and making the code easier to test.
- performed in its own function/method to ensure we can cover it with unit tests.